contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py

```diff
@@ -115,8 +115,7 @@ class AzureLogHandler(TransportMixin, BaseLogHandler):
 
     def __init__(self, **options):
         self.options = Options(**options)
-        if not self.options.instrumentation_key:
```

Comment:
Is it necessary to check null for ikey twice since it'll be checked in validate_key first?

Reply:
It is not necessary. The `if not self.options.instrumentation_key:` is a deleted line :)